assumptions/tools/ahem-generate-table.py

```diff
@@ -18,7 +18,7 @@ def grouper(n, iterable):
     iterable = iter(iterable)
     return iter(lambda: list(itertools.islice(iterable, n)), [])
 
-ttf = TTFont("../../css/fonts/ahem/ahem.ttf")
+ttf = TTFont("../../fonts/Ahem.ttf")
```

Comment:
I would prefer absolute paths if possible.

Reply:
How do we get an absolute path to the root of the repo?

Reply:
`/fonts/Ahem.ttf`?

Reply:
That won't work. This is a Python script. That's resolved relative to the filesystem root and not the repo root. As you said in IRC, you're "being dumb". Ignoring and merging. :)